app/assets/javascripts/events.js.coffee

```diff
@@ -6,9 +6,23 @@ jQuery ->
      timepicker:false,
      format:'d.m.Y'
   });
-  jQuery('.invites input').on 'change', ->
+  jQuery('.list-sum input').on 'change', ->
+    list_id = jQuery(this).attr('id');
+    list_sum = Math.abs(jQuery(this).val());
+    jQuery(this).val(list_sum);
+    jQuery.ajax({
```

Comment:
content-type not specified so controller won't know how to respond and jQuery defaults will be used